### PR TITLE
DAOS-7621 test: Apply functional test tags/flags to dfuse.

### PR DESCRIPTION
**Pull request by ashleypittman**
Quick-Functional: true
Test-tags: dfuse

Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: src/tests/ftest/ior/ior_small.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def test_ior_small(self):
 
         :avocado: tags=all,pr,daily_regression
         :avocado: tags=hw,large
-        :avocado: tags=daosio,checksum,iorsmall,mpich
+        :avocado: tags=daosio,checksum,iorsmall,mpich,dfuse
```
Comment (daltonbohning) [inline src/tests/ftest/ior/ior_small.py:39]: Another one that was not formatted correctly before. I would think 39 and 40 here should be:
```
:avocado: tags=daosio,checksum,mpich,dfuse,DAOS_5610
:avocado: tags=iorsmall
```

**Review by daltonbohning — APPROVED** [review]
The tags appear to be functional - just some not 100% standardly formatted. I'll let @sylviachanoiyee decide whether that should require another PR run. In general, we still need to do more scrubbing of the test tags, so maybe my comments would be handled elsewhere.

**Comment by ashleypittman** [comment]
> The tags appear to be functional - just some not 100% standardly formatted. I'll let @sylviachanoiyee decide whether that should require another PR run. In general, we still need to do more scrubbing of the test tags, so maybe my comments would be handled elsewhere.

Done, and I've merged with master to pick up latest fixes as well.

**Review by daltonbohning — DISMISSED** [review]
Thanks!

**Comment by sylviachanoiyee** [comment]
I cannot see how the changes in this PR would cause failures in a bunch of datamover tests for failing to create mountpoint
2021-06-22 11:27:16,890 general_utils    L0409 INFO | Results:
2021-06-22 11:27:16,890 general_utils    L0413 INFO |   wolf-100vm1: exit_status=0, interrupted=False:
2021-06-22 11:27:16,890 general_utils    L0415 INFO |     ext2/ext3
2021-06-22 11:27:17,093 general_utils    L0408 INFO | Command: mkdir -p /tmp/daos_dfuse
2021-06-22 11:27:17,093 general_utils    L0409 INFO | Results:
2021-06-22 11:27:17,093 general_utils    L0413 INFO |   wolf-100vm3: exit_status=1, interrupted=False:
2021-06-22 11:27:17,093 general_utils    L0415 INFO |     mkdir: cannot create directory â€˜/tmp/daos_dfuseâ€™: File exists
2021-06-22 11:27:17,094 dfuse_test_base  L0066 ERROR| Dfuse command dfuse --mountpoint /tmp/daos_dfuse failed on hosts wolf-100vm[1,3]

but because fn vm failed, fn hw did not run

**Comment by ashleypittman** [comment]
> I cannot see how the changes in this PR would cause failures in a bunch of datamover tests for failing to create mountpoint
> 2021-06-22 11:27:16,890 general_utils L0409 INFO | Results:
> 2021-06-22 11:27:16,890 general_utils L0413 INFO | wolf-100vm1: exit_status=0, interrupted=False:
> 2021-06-22 11:27:16,890 general_utils L0415 INFO | ext2/ext3
> 2021-06-22 11:27:17,093 general_utils L0408 INFO | Command: mkdir -p /tmp/daos_dfuse
> 2021-06-22 11:27:17,093 general_utils L0409 INFO | Results:
> 2021-06-22 11:27:17,093 general_utils L0413 INFO | wolf-100vm3: exit_status=1, interrupted=False:
> 2021-06-22 11:27:17,093 general_utils L0415 INFO | mkdir: cannot create directory â€˜/tmp/daos_dfuseâ€™: File exists
> 2021-06-22 11:27:17,094 dfuse_test_base L0066 ERROR| Dfuse command dfuse --mountpoint /tmp/daos_dfuse failed on hosts wolf-100vm[1,3]
> 
> but because fn vm failed, fn hw did not run

This is new, but the only thing I can think of is that a previous test has not shut down properly but I can't see any evidence of this.

**Comment by sylviachanoiyee** [comment]
I restarted the test, just to be sure.

**Comment by ashleypittman** [comment]
> I restarted the test, just to be sure.

I'd already done that so it's run three times now.

1st time it had 7 failures, then 5, then 6.

For now I've merged with master and no test tags to see how that does, when that's complete I'll try it with the dfuse test tag again and try to diagnose.

**Review by daltonbohning — APPROVED** [review]
I don't think the commit pragma is carried-over from the previous commit, but otherwise LGTM.

**Review by sylviachanoiyee — APPROVED** [review]

**Pull request merged**